yearn/prices/chainlink.py

```diff
@@ -63,7 +66,7 @@ def get_price(self, asset, block=None):
             return None
         try:
             return self.get_feed(asset).latestAnswer(block_identifier=block) / 1e8
-        except ValueError:
+        except (ValueError, KeyError):
```

Comment:
I've changed this because during refactoring my code was incorrectly calling the oracle. Removed it again in favour of more explicit exceptions and less obscurity. Thanks for pointing this out!